index.bs

```diff
@@ -1341,7 +1341,7 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
         Note: An [=RP ID=] is based on a [=host=]'s [=domain=] name. It does not itself include a [=scheme=] or [=port=], as an [=origin=] does. The [=RP ID=] of a [=public key credential=] determines its <dfn>scope</dfn>. I.e., it <dfn>determines the set of origins on which the public key credential may be exercised</dfn>, as follows:
 
             - The [=RP ID=] must be equal to the [=determines the set of origins on which the public key credential may be exercised|origin=]'s [=effective domain=], or a [=is a registrable domain suffix of or is equal to|registrable domain suffix=] of the [=determines the set of origins on which the public key credential may be exercised|origin=]'s [=effective domain=].
-            - The [=determines the set of origins on which the public key credential may be exercised|origin=]'s [=scheme=] must be `https`.
+            - The [=determines the set of origins on which the public key credential may be exercised|origin=]'s [=scheme=] must be `https`, or considered a [=secure contexts|secure context=].
```

Comment:
Bikeshed should [automatically resolve pluralization](https://speced.github.io/bikeshed/#conjugating), so different linking and visible texts should not be necessary here.

```suggestion
            - The [=determines the set of origins on which the public key credential may be exercised|origin=]'s [=scheme=] must be `https`, or considered a [=secure context=].
```